industrial_taxonomy/pipeline/official/gdp/flow.py

```diff
@@ -0,0 +1,69 @@
+"""Fetch GDP data"""
+
+from typing import Dict, List, Union
+import pandas as pd
+
+from metaflow import FlowSpec, project, step
+
+URL = (
+    "https://www.ons.gov.uk/file?uri=/economy/grossdomesticproductgdp/datasets/",
+    "regionalgrossdomesticproductlocalauthorities/1998to2019/",
+    "regionalgrossdomesticproductlocalauthorities.xlsx",
```

Comment:
If the comma separation from these is removed then no need for self.url = "".join(URL) below